compiler/rustc_session/src/config/cfg.rs

```diff
@@ -176,6 +176,8 @@ pub(crate) fn default_configuration(sess: &Session) -> Cfg {
     // NOTE: These insertions should be kept in sync with
     // `CheckCfg::fill_well_known` below.
 
+    ins_none!(sym::autodiff_fallback);
```

Comment:
This shouldn't be insta stable, it should be at least gated behind nightly compiler.
```suggestion
    if sess.is_nightly_build() {
        ins_none!(sym::autodiff_fallback);
    }
```

----

Please also follow all the steps regarding a new cfg as defined in the top of this file (as well as the tests files):
https://github.com/rust-lang/rust/blob/e26b02a2607bf9ab1b2f16cd877213ec4d5161a0/compiler/rustc_session/src/config/cfg.rs#L10-L21